lib/settings/presentationSettings.js

```diff
@@ -0,0 +1,129 @@
+'use strict';
+
+var hooks = require('../hooks/hooks.js');
+
+
+var Promise     = require("bluebird")  
+  , coroutine   = Promise.coroutine
+  , fs = require("fs");
+
+var Exercise = db.model('Exercise');
+var Slideshow = db.model('Slideshow');
+var Setting = db.model('Setting');
+var ObjectId = require('mongoose').Types.ObjectId;
+
+Promise.promisifyAll(fs);
+
+
+module.exports = {
+
+  // Get settings of exercises and transform them to fit the format with dust template.
```

Comment:
// Get settings of exercises and transform them to fit the format with dust template. ->  // Get settings of exercises and transform them to fit the format used in the dust template.